### DISCUSSION TITLE
Can I used a Harman License with Flex 4.16.1 + AIR SDK 32.0?

### DISCUSSION THREAD

**Discussion by ULSashido** [airsdk/Adobe-Runtime-Support#1945] [Q&A]
I am trying to purchase a License from Harman, though I am not sure if I can use the license on AIR 32 or am I obliged to used AIR 33.1 in order for the Harman Splash to be removed prior to loading the desktop app?

_Reason for using AIR 32, is because I am still using Flash Builder (on a MacOS Mohave dedicated to developing Flex/AIR Desktop Apps), and it doesn't seem possible to Export a Release Build with AIR 33.1 with Flash Builder._

**Comment by ajwfrost** [comment]
If you've got AIR 32 then you don't need any license from us, that one is provided by Adobe under their free license agreement. So if you're using AIR 32 there won't be any splash screen or similar..

If you need the AIR 33+ functionality (e.g. Android 64-bit support, support for latest macOS/iOS versions etc) then you'd need to use our version of the AIR SDK with the splash screen appearing for "free tier" eligible people, or paying for an SDK license..

There are some issues when using Flash Builder with AIR 33.1 e.g. you need to update the JRE version, plus I think some other restrictions we've found e.g. you can't have a folder defined for the output. Personally I would suggest using FB+AIR32 in your existing set-up while you can but then move to something else like IntelliJ IDEA, Moonshine, VS Code, etc... Flash Builder isn't supported and so can't make use of some of the AIR SDK updates..

thanks

**Reply by ULSashido** [reply]
If I export release with AIR 32 SDK, and then run the `.air` App on a Mac with AIR Runtime 33.1, the splash screen does appear, unless I am able to install the AIR 32 runtime? (which I cannot seem to find the AIR 32 runtime).

**Reply by ajwfrost** [reply]
Ah, yes, good point. So you could try exporting a "bundle" from the AIR 32 SDK (if you've got a mac), and if you can code-sign/notarize it, it should work with the captive AIR 32 runtime. But the shared AIR 32 runtime installers aren't available any more...

The AIR 33.1 shared runtime will display a splash screen for all "old" apps, or basically, for anything that wasn't packaged up with a commercial version of the AIR 33+ SDK...

thanks
